### ISSUE TITLE
upload image solution

### ISSUE THREAD

**Issue by august-zou** [Sottti/OkHttpVolleyGson#1]
if the demo add the upload images or files solution,it will be perfect :)

**Comment by jackTang11** [comment]
if the demo add the upload images or files solution,it will be perfect :)

**Comment by ateek** [comment]
yeah i think so , the upload image part will be cool if it added , thanks btw for the great example
